fs-storage/src/base_storage.rs

```diff
@@ -1,7 +1,9 @@
 use data_error::Result;
+use jnix_macros::{FromJava, IntoJava};
```

Comment:
Why is `jnix_macros` preferable over `jnix` which also exports `jnix_macros` under a feature flag?